src/containers/WelcomePage/components/SavedSearch.tsx

```diff
@@ -32,14 +37,18 @@ const StyledSearch = styled.div`
   align-items: center;
 `;
 
-const SavedSearch = ({ deleteSearch, search, index }: Props) => {
+const SavedSearch = ({ deleteSearch, search, index, userData }: Props) => {
   const { t, i18n } = useTranslation();
   const { taxonomyVersion } = useTaxonomyVersion();
   const locale = i18n.language;
   const [searchUrl, searchParams] = search.split('?');
 
-  const searchObject = transformQuery(queryString.parse(searchParams));
-  const subject = searchObject['subjects'] || '';
+  const [searchObject, isFavorite] = useMemo(() => {
+    const obj = transformQuery(queryString.parse(searchParams));
+    const isFav = obj.subjects === FAVOURITES_SUBJECT_ID;
```

Comment:
Kan like gjerne bruke isFavourite i stedet for en kortversjon. Eller er det for å unngå å måtte velge mellom isFavorite og isFavourite? :)